applications/aero_acoustic/co_rotating_vortex_pair/application.h

```diff
@@ -636,6 +636,44 @@ class Application : public ApplicationBase<dim, Number>
 
 namespace AeroAcoustic
 {
+template<int dim>
+class BlendInFunction : public Utilities::SpatialAwareFunction<dim>
+{
+public:
+  BlendInFunction(double const blend_in_start, double const blend_in_end)
+    : Utilities::SpatialAwareFunction<dim>(1, 0.0), start(blend_in_start), end(blend_in_end)
+  {
+  }
+
+  double
+  value(dealii::Point<dim> const &, unsigned int const) const final
+  {
+    return compute_time_factor();
```

Comment:
Do I understand the code correctly that we will never run into this function, because `varies_in_space()` is implemented here in this example such that it returns `false`?

If so, I consider this implementation of `Function::value()` irritating. Users who want to implement something similar will not know what they have to write here in my opinion. So I would leave this function empty with a short comment like "will not be used for this example" or similar.